src/redux/epics/event.epics.js

```diff
@@ -4,12 +4,13 @@ import { Observable } from 'rxjs';
 import 'rxjs/add/operator/mergeMap';
 
 const BASE_ENDPOINT = `https://www.eventbriteapi.com/v3/events/search/?token=${EVENTBRITE_API_KEY}&location.address=toronto&categories=`;
+const reCategoryPath = /^.*\/category\/\d\d\d$/;
 
 export const returnEventActionOnLocationChange = (action$) =>
```

Comment:
I thought about something similar, but this epic doesn't make the api call or get the events, it just returns the getEvents action.